storage/schemadb/src/lib.rs

```diff
@@ -204,26 +185,15 @@ impl DB {
             );
         }
 
-        let mut db_opts = DBOptions::new();
+        let mut db_opts = rocksdb::Options::default();
+        db_opts.create_if_missing(true);
+        db_opts.create_missing_column_families(true);
 
         // For now we set the max total WAL size to be 1G. This config can be useful when column
         // families are updated at non-uniform frequencies.
         db_opts.set_max_total_wal_size(1 << 30);
 
-        // If db exists, just open it with all cfs.
-        if db_exists(path.as_ref()) {
-            return DB::open_cf(db_opts, &path, column_families);
-        }
-
-        // If db doesn't exist, create a db first with all column families.
-        db_opts.create_if_missing(true);
-
-        let mut db = DB::open_cf(db_opts, path, vec![DEFAULT_CF_NAME])?;
-        column_families
-            .into_iter()
-            .filter(|cf_name| *cf_name != DEFAULT_CF_NAME)
-            .map(|cf_name| db.create_cf(cf_name))
```

Comment:
Curious why did we need to create_cf explicitly before?

Reply:
The library we were using does not have the `create_missing_column_families` option. Let me double check how this one is implemented.

Reply:
https://github.com/facebook/rocksdb/blob/243852ec15f96de760df8612b6ae9f4664297e55/include/rocksdb/options.h#L385